### PR TITLE
fix: kiuwan reports

### PR DESCRIPTION
**Pull request by Matiasnickolas**
fix alerts reported by kiuwan

- [x] Remove unnecesary commas
- [x] Use https
- [x] Use hpp
- [x] Use cors
- [x] Add csrf protection
- [x] Limit max sockets connections
- [x] Use helmet
- [x] Refactor too large functions

connect to port:

image

connect to https:
image

poll:

image

image

sale:

image

last sale:

image

load Keys: 

image

change to normal mode:

image

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+var AutoLaunch = require('auto-launch');
```
Comment (mastudillot) [inline src/index.ts]: Use const instead of var

File: src/server/app.ts [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
+const https = require('https');
+const fs = require('fs');
+const socket = require('socket.io');
 import pos from '../pos';
 import windowManager from "../classes/window-manager";
 import posHandler from './poshandler';
+
+const app = express();
+
+const crtFolder: string = process.env.NODE_ENV === "production" ?
```
Comment (mastudillot) [inline src/server/app.ts:19]: These paths are hard to understand, can you find a way to make them more readable?

File: src/server/app.ts [hunk 3]
```diff
@@ -12,6 +12,10 @@ const crtFolder: string = process.env.NODE_ENV === "production" ?
     path.join(__dirname, '..', '..', '..', 'crt/') :
     path.join(__dirname, '..', '..', 'crt/');
 
+if(!/[A-Z][A-Z0-9_]*/i.test(crtFolder)) {
```
Comment (mastudillot) [inline src/server/app.ts]: Is this necessary?

Comment (Matiasnickolas) [reply src/server/app.ts]: this is a kiuwan recomendation

**Review by mastudillot — APPROVED** [review]
LGTM

**Review by isabellaahc — APPROVED** [review]
lgtm 🎉

**Pull request merged**